api/server/server_test.go

```diff
@@ -11,7 +11,9 @@ import (
 )
 
 func TestMiddlewares(t *testing.T) {
-	cfg := &Config{}
+	cfg := &Config{
+		Version: "1.11",
```

Comment:
Maybe you can check that it's from config now below. And let's use something that won't be in dockerversion.Version like `0.1omega2`